.github/flakybot.yaml

```diff
@@ -0,0 +1 @@
+issuePriority: p2
```

Comment:
Please add a license header similar to
https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/flakybot.yaml

Reply:
Thanks @parthea for the feedback! Do you know if the license header will be auto generated? Or we should add that manually?

Reply:
It should be added manually

Reply:
Sounds good! Added the license header.